CHANGELOG.md

```diff
@@ -1,5 +1,100 @@
 # Changelog
 
+## 0.3.0
+
+This release stabilizes all preview functionality in uv. In brief, this means that:
```

Comment:
What about "This release stabilizes a set of new features in uv. See our [blog post](#) for an overview. All features previously in preview are now stable. In brief, this means that:"? I want to avoid the impression that you needed to have followed the preview development for the stable changelog to make (something that other python projects have done)

Reply:
This changed now — but yes we can and should link to the blog post.